libcxxabi/test/vendor/ibm/cond_reg_restore.pass.cpp

```diff
@@ -10,7 +10,7 @@
 // on AIX. Option -O3 is required so that the compiler will re-use the value
 // in the condition register instead of re-evaluating the condition expression.
 
-// REQUIRES: target=powerpc{{(64)?}}-ibm-aix
+// REQUIRES: target={{.+}}-aix{{.*}}
```

Comment:
Why drop the vendor in the vendor specific directory?

Reply:
> Why drop the vendor in the vendor specific directory?

IBM is the only vendor shipping the AIX OS so I think the vendor part is unnecessary. This is similar to {{.+}}-zos{{.*}}.